src/store/mutations/mutations-model.ts

```diff
@@ -80,7 +80,6 @@ export const mutateCertifyState = (state: StateIF, certifyState: CertifyIF) => {
 }
 
 export const mutateCertifyStateValidity = (state: StateIF, validity: boolean) => {
-  state.stateModel.certifyState.valid = validity
```

Comment:
What was this used for? Corrections maybe?

Reply:
Please test corrections after your changes.

Reply:
Its called from CertifySection.vue which is common for Correction and Alteration. This was tracking only the checkbox value